.stylelintrc

```diff
@@ -1,5 +1,7 @@
 {
+  "extends": "stylelint-config-standard",
   "rules": {
-    "selector-class-pattern": "mapboxgl-[a-z-]+"
+    "indentation": 4,
+    "selector-class-pattern": "mapbox(gl)?-[a-z-]+"
```

Comment:
@mollymerp I've done that, just in a different location: the .stylelintrc file now has a more flexible selector pattern so that those disable comments aren't needed in the CSS.